hikari/impl/entity_factory.py

```diff
@@ -1993,8 +1993,8 @@ def deserialize_command_interaction(
             id=snowflakes.Snowflake(payload["id"]),
             type=base_interactions.InteractionType(payload["type"]),
             guild_id=guild_id,
-            guild_locale=locales.Locale(payload.get("guild_locale", "en-US")),
-            locale=locales.Locale(payload["locale"]),
+            guild_locale=locales.Locale(payload.get("guild_locale")) if "guild_locale" in payload else None,
```

Comment:
```suggestion
            guild_locale=locales.Locale(payload.get("guild_locale", "en-US")),
```

Reply:
None will be used for commands used in dms. Discord will always send `en-US` when in guild:
```py
    guild_locale: typing.Optional[typing.Union[str, locales.Locale]] = attr.field(eq=False, hash=False, repr=True)
    """The preferred language of the guild this command interaction was triggered in.

    This will be `builtins.None` for command interactions triggered in DMs.

    !!! note
        This value can usually only be changed if `COMMUNITY` is in `hikari.guilds.Guild.features`
        for the guild and will otherwise default to `en-US`.
    """
```